client/state/audio/middleware.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Internal dependencies
+ */
+import {
+	HAPPYCHAT_RECEIVE_EVENT,
+} from 'state/action-types';
+
+const isAudioSupported = () => typeof window === 'object' && typeof window.Audio === 'function';
+
+export const playSound = src => {
+	if ( ! isAudioSupported() ) {
+		return;
+	}
+
+	const audioClip = new window.Audio( src );
+	audioClip.play();
+};
+
+export const onHappyChatMessage = ( dispatch, { event } ) => {
+	event && event.source !== 'customer' && playSound( '/calypso/audio/chat-pling.wav' );
```

Comment:
so I was confused on which messages should trigger a ding. I think naming could help out a bit here. something like `playNewOperatorMessageDing` (probably need a better name than that) maybe